docroot/modules/custom/mass_translations/mass_translations.routing.yml

```diff
@@ -0,0 +1,23 @@
+mass_translations.controller_translations:
+  path: '/node/{node}/translations'
+  defaults:
+    _controller: '\Drupal\mass_translations\Controller\NodeTranslationsController::content'
+    _title: 'Translations'
+  options:
+    _node_operation_route: TRUE
+  requirements:
+    _permission: 'view Translations tab'
+    _custom_access: '\Drupal\mass_translations\Controller\NodeTranslationsController::access'
+mass_translations.controller_media_translations:
+  path: '/media/{media}/translations'
+  defaults:
+    _controller: '\Drupal\mass_translations\Controller\MediaTranslationsController::content'
+    _title: 'Translations'
+  options:
+    _admin_route: TRUE
```

Comment:
Any particular reason why this is an admin route but the one above is not?

Reply:
The one above is using _node_operation_route. I had to jog my memory, but _node_operation_route sets _admin_route